vyper/utils.py

```diff
@@ -45,9 +45,16 @@ def __len__(self):
     def first(self):
         return next(iter(self))
 
+    def pop(self):
```

Comment:
Maybe popleft()? Since it's lifo

Reply:
popitem() is also lifo

Reply:
> popitem() is also lifo

right, `pop()` (on lists) is FIFO though

Reply:
```suggestion
    def popleft(self):
```

Reply:
Seems to me that `pop()` is also LIFO on lists... :confusion:

Reply:
idts, there is `pop()` for lists which is FIFO, and `popleft()` for deques which is LIFO

Reply:
whoops, i switched FIFO and LIFO